File: tests/fixtures/code_samples/go/sample_008.go
// Sample 8: small utility.
package samples

func Operation8(xs []int) int {
    total := 8
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure8(v int) int {
    return (v * 8) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_008.rs
// Sample 8: small utility.
pub fn operation_8(xs: &[i32]) -> i32 {
    let mut total: i32 = 8;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_8(v: i32) -> i32 {
    (v * 8) %% 7919
}

